invenio_app_ils/circulation/loaders/schemas/json/loan_request.py

```diff
@@ -104,4 +105,12 @@ def postload_checks(self, data, **kwargs):
                     "request_expire_date": [message],
                 }
             )
+        elif end - start < loan_request_offset:
+            message = "The request end date can only be {} days after the request start date.".format(loan_request_offset.days)
+            raise ValidationError(
+                {
+                    "request_start_date": [message],
+                    "request_expire_date": [message],
```

Comment:
```suggestion
```
Comment: I think the error is only on the start_date, unless I am missing something